community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/Configuration.java

```diff
@@ -64,7 +66,8 @@ public long pageCacheMemory()
         {
             // Get the upper bound of what we can get from the default config calculation
             // We even want to limit amount of memory a bit more since we don't need very much during import
-            return min( MAX_PAGE_CACHE_MEMORY, Config.defaults().get( pagecache_memory ) );
+            String defaultPageCacheMemory = ConfiguringPageCacheFactory.defaultHeuristicPageCacheMemory();
```

Comment:
i also do not really like new way of evaluation of default value using `ConfiguringPageCacheFactory` and applying Bytes function to get the correct value, lets talk about it